Library/Library/BookCopies.aspx

```diff
@@ -0,0 +1,29 @@
+﻿<%@ Page Title="" Language="C#" MasterPageFile="~/Site.Master" AutoEventWireup="true" CodeBehind="BookCopies.aspx.cs" Inherits="Library.BookCopies" %>
+<%@ import namespace="System.Data" %>
+<asp:Content ID="Content1" ContentPlaceHolderID="main" runat="server">
+    <asp:repeater id="Books" runat="server" itemtype="DataRow">
```

Comment:
Be really mindful of the IDs that you use for your controls. For example, this repeater control contains a list of "book copies" (or "item copies") which is distinctly different than "books" (or "items"). Using an ID value that accurately describes the data the control is bound too will help code reviewers understand what your code is doing.